syntax/dashboard.sublime-settings

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Noboy should use this syntax file. It is just a base for branch.sublime-syntax, rebase.sublime-syntax, status.sublime-syntax and tags.sublime-syntax. https://github.com/deathaxe/GitSavvy/blob/fd7ad09d1782cf8c43bc82ef87baa22e55830a7d/syntax/rebase.sublime-syntax#L9-L9

Reply:
Ok, so we can remove this settings file.